UnionFind/NumberOfProvinces.py

```
@@ -0,0 +1,30 @@
+class Solution:
+      def __init__(self):
+          self.array=[]
+      def find(self,node):
+          if self.array[node]==node:
+             return node 
+          return self.find(self.array[node]) 
+      def union(self,node_1,node_2):
```

Flake8 (7.2.0)

[error] expected 1 blank line, found 0

(E301)